pkg/build/builder/util.go

```diff
@@ -55,11 +57,60 @@ func getDockerNetworkMode() stiapi.DockerNetworkMode {
 	defer file.Close()
 
 	if id := readNetClsCGroup(file); id != "" {
-		return stiapi.NewDockerNetworkModeContainer(id)
+		return s2iapi.NewDockerNetworkModeContainer(id)
 	}
 	return ""
 }
 
+// GetCGroupLimits returns a struct populated with cgroup limit values gathered
+// from the local /sys/fs/cgroup filesystem.  Overflow values are set to
+// MAX_INT_64.
+func GetCGroupLimits() (*s2iapi.CGroupLimits, error) {
```

Comment:
and this is not a Getter, it's a function that's taking an action to go read values.  I think Get is an appropriate name given that behavior.  it's not just returning a stored value.